cmd/build_test_matrix/main_test.go

```diff
@@ -17,13 +17,16 @@ const (
 )
 
 func TestGetGithubActionMatrixForTests(t *testing.T) {
+	t.Parallel()
```

Comment:
please keep the pr scoped to only updating to 0.50. This pr is now a blocker for full cosmwasm integration

Reply:
eh?  well that is confusing but I def need to revert that commit.  

Thank you for your review.